### PR TITLE
To do

### PR DESCRIPTION
**Pull request by Sharmaryan**
user can
-  add task
- delete task
- edit task
- update task

### CODE REVIEW

File: src/components/Task/Task.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+.task-title {
+    word-break: break-word;
+    text-align: left;
+    width: 70%;
+}
+
+.task-buttons {
+    margin-left: auto;
+
```
Comment (Kothalkarpriya) [inline src/components/Task/Task.css:17]: remove unnecessary space from code

**Review by Kothalkarpriya — APPROVED** [review]
You did good work , you can merge

File: src/App.css [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+  --black : black;
+}
 .App {
   text-align: center;
-}
-
-
-
-
+  background-color: rgb(159, 240, 159);
```
Comment (raghav6202) [inline src/App.css:13]: You can remove extra space here

**Comment by raghav6202** [comment]
Clean code. Would be better if there was a deploy preview available.

**Review by raghav6202 — APPROVED** [review]

**Pull request merged**